src/js/browser-plugin.js

```diff
@@ -1525,18 +1580,27 @@ function updateLabelEvaluation(targetButton, tweetInfo, agreement) {
     updateLabelEvaluationAgg(auxPrevious, tweetInfo, pluginCache[tweetInfo.id].feedback.user_feedback, "remove", 1);
   }
   targetButton.classList.add("coinformToolbarFeedbackAfterClick");
+  // eslint-disable-next-line camelcase
   pluginCache[tweetInfo.id].feedback.user_feedback = agreement;
 }
 
 function updateLabelEvaluationAgg(targetButton, tweetInfo, agreement, operation, num) {
   let totalNum = 0;
   if (pluginCache[tweetInfo.id].feedback[`total_${agreement}`] != undefined) {
-    totalNum = pluginCache[tweetInfo.id].feedback[`total_${agreement}`];
+    totalNum = parseInt(pluginCache[tweetInfo.id].feedback[`total_${agreement}`]);
   }
-  if (operation == 'add') totalNum = totalNum + num;
-  else if (operation == 'remove') totalNum = totalNum - num;
-  else if (operation == 'update') totalNum = num;
-  if (totalNum >= 1) {
+  if (operation == 'add') totalNum = totalNum + parseInt(num);
```

Comment:
can do a += here instead of the long syntax

Reply:
I agree, I'll update that